src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -446,6 +465,43 @@
       allow_any_instance_of(Project).to receive(:number_of_build_problems).and_return(0)
     end
 
+    context 'with bdep and jobs' do
+      let(:bdep_url) { 'http://localhost:3200/build/home:tom/repository_1/x86_64/_builddepinfo' }
```

Comment:
Nevermind, this requires a build, which we do not really support in the test backend...